src/common/data_provider/src/sharedDefs.h

```diff
@@ -50,7 +46,6 @@ enum OSPlatformType
     LINUX,
     BSDBASED,
```

Comment:
Also remove this line.

Reply:
This may impact on macOS compilation, we can discuss this later.